metrics/register/session_manager.go

```diff
@@ -0,0 +1,19 @@
+package register
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rancher/norman/metrics"
+)
+
+func init() {
+	prometheus.MustRegister(metrics.TotalAddWS)
```

Comment:
Is there any way to gate these `MustRegister` behind the env var?

Reply:
I don't have particular opinion about how/where we evaluated MustRegister. 
We can move them to metrics/register.go.

The reason why I put them in metrics/register/session_manager.go is that I gave a respect to the way 
to initialize "GenericController" metrics which is implemented in Norman already.